docs/pages/usage/integrations/index.rst

```diff
@@ -6,6 +6,7 @@
   auto-formatters.rst
   nitpick.rst
   flakeheaven.rst
```

Comment:
please, remove this file

Reply:
`flakeheaven.rst` file too?

Reply:
I mean `flakeheaven.rst` :)

Reply:
Removed in [commit](https://github.com/wemake-services/wemake-python-styleguide/pull/2975/commits/f213364e6cc34a9d0335581817e840b72b41bfa1)